skytap/environment.go

```diff
@@ -234,7 +245,7 @@ func (s *EnvironmentsServiceClient) Create(ctx context.Context, request *CreateE
 		SuspendAtTime:   request.SuspendAtTime,
 		ShutdownOnIdle:  request.ShutdownOnIdle,
 		ShutdownAtTime:  request.ShutdownAtTime,
-		Runstate:        &runstate,
+		Runstate:        runstate,
```

Comment:
does the logic of starting the environment belong to terraform more than here?  Can we add a comment that explains why the status depends on the environment?